py3-ansible-core/resolvlib.patch

```diff
@@ -0,0 +1,166 @@
+From d4085a56e00a6f9059e7df6e4a16eef4f5737501 Mon Sep 17 00:00:00 2001
+From: s-hertel <19572925+s-hertel@users.noreply.github.com>
+Date: Fri, 1 Nov 2024 11:49:04 -0400
+Subject: [PATCH 1/2] Bump ansible-galaxy's resolvelib requirement upperbound
+ to 1.2.0
+
+Test against the newest resolvelib release 1.1.0
+
+Only test the oldest supported resolvelib version, latest minor release,
+and releases that offer additional coverage
+---
+ changelogs/fragments/update-resolvelib-lt-1_2_0.yml   |  2 ++
+ lib/ansible/galaxy/dependency_resolution/providers.py |  2 +-
+ requirements.txt                                      |  2 +-
+ .../targets/ansible-galaxy-collection/vars/main.yml   | 11 +++++------
+ test/lib/ansible_test/_data/requirements/ansible.txt  |  2 +-
+ 5 files changed, 10 insertions(+), 9 deletions(-)
+ create mode 100644 changelogs/fragments/update-resolvelib-lt-1_2_0.yml
+
+diff --git a/changelogs/fragments/update-resolvelib-lt-1_2_0.yml b/changelogs/fragments/update-resolvelib-lt-1_2_0.yml
+new file mode 100644
+index 00000000000000..461f436339b4c5
+--- /dev/null
++++ b/changelogs/fragments/update-resolvelib-lt-1_2_0.yml
+@@ -0,0 +1,2 @@
++minor_changes:
++  - ansible-galaxy - support ``resolvelib >= 0.5.3, < 1.2.0`` (https://github.com/ansible/ansible/issues/84217).
```

Comment:
Because I see that it's been patched to `2.0.0` in the upstream and [merged](https://github.com/ansible/ansible/pull/84218/files#diff-065abed58d290df207524c494d40352b8dc2fb11ecb86b3e58553c9a3888656fR15) to the `devel` branch,
even if not setting the update to manual, maybe we should highlight this in case the patch will no longer apply in the next release?

Reply:
I think this will be removed as part of automation in the next update.